### PR TITLE
Javascript/javascript1/week3

### PR DESCRIPTION
**Pull request by rajanibusani**
added homework for week3

### CODE REVIEW

File: javascript/javascript1/week3/index.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+// 1 year = approx. 8766 hours
+const averageLifeSpanInHours = averageLifeSpan * 8766;
+let total =0;
+function seriesDurationInLife() {
+
+  for (let i=0; i<seriesDurations.length; i++){
+  // changing the days, and minutes to hours
+  const totalDurationInHours = (seriesDurations[i].days*24)+ seriesDurations[i].hours+ (seriesDurations[i].minutes/60);
+  
+  let percentageInLifeSpan = (totalDurationInHours/averageLifeSpanInHours)*100;
```
Comment (aina21) [inline javascript/javascript1/week3/index.js:72]: const

File: javascript/javascript1/week3/index.js [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+saveNote("pick up online-order",4)
+
+console.log(notes); // [{content: 'Pick up groceries', id: 1}, {content: 'Do laundry', id: 2}, {content: 'Pay bills', id: 3},{content: "pick up online-order", id: 4}]
+
+//Get a note with id value
+
+function getNote(id) {
+
+  for (let i=1; i<= notes.length; i++){
+    if(i===id){
```
Comment (aina21) [inline javascript/javascript1/week3/index.js:105]: do you mean :
if(notes[i].id === id)

File: javascript/javascript1/week3/phoneusage.html [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    <body>
+    
+    <script>
+    //smartphone usage
+//adding activities on the phone with todays's date.
+"use strict"
+const activities = [];
+function addActivity(activity, duration){
+  
+    let date= new Date();
```
Comment (aina21) [inline javascript/javascript1/week3/phoneusage.html:19]: const

File: javascript/javascript1/week3/phoneusage.html [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+    let todaysDate =date.toLocaleDateString();
```
Comment (aina21) [inline javascript/javascript1/week3/phoneusage.html:20]: const

File: javascript/javascript1/week3/phoneusage.html [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+{date: "11/11/2020", activity: "News", duration: 10}*/
+
+//showStatus(activities); setting a limit for smartphone usage
+
+let totalTime = 0;
+const usageLimit = 120;
+function showStatus(activities){
+  if(activities.length===0){
+    console.log("Add someactivities before calling showStatus.")
+  }else {
```
Comment (aina21) [inline javascript/javascript1/week3/phoneusage.html:43]: do you need else?

File: javascript/javascript1/week3/uniquefeature.html [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+<html lang="en">
+    <head>
+        <title>
+            notes saving app with extra feature
+        </title>
+        <meta charset="utf-8">               
+    </head>
+    <body>
+        <script>
+          "use strict"
```
Comment (aina21) [inline javascript/javascript1/week3/uniquefeature.html:11]: Add js code in a separate file.

File: javascript/javascript1/week3/uniquefeature.html [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+console.log("Savenote app with my unique feature.")
+/*with in savenote app, i want add check mark which will help to separate finished and unfinished tasks*/ 
+const notes = [];
+const finishedTasks=[];
+const unfinishedTasks=[];
+function saveNote(content, id,check) {  
+   notes.push({content:content, id:id});
+
+  if (check ===true){
```
Comment (aina21) [inline javascript/javascript1/week3/uniquefeature.html:20]: if(check)

File: javascript/javascript1/week3/uniquefeature.html [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+      finishedTasks.push({content:content, id:id});    
+    
+  }else if (check === false|| check === undefined){
```
Comment (aina21) [inline javascript/javascript1/week3/uniquefeature.html:23]: else is enough you don't need other condition

**Review by aina21 — CHANGES_REQUESTED** [review]
good job 👍

File: javascript/javascript1/week3/index.js (hunk 1 above)
Comment (rajanibusani) [reply javascript/javascript1/week3/index.js:72]: changes to const ida...

File: javascript/javascript1/week3/phoneusage.html (hunk 3 above)
Comment (rajanibusani) [reply javascript/javascript1/week3/phoneusage.html:19]: changed to const

File: javascript/javascript1/week3/phoneusage.html (hunk 5 above)
Comment (rajanibusani) [reply javascript/javascript1/week3/phoneusage.html:43]: Ya...true, i dont need else here, removed it..tq

File: javascript/javascript1/week3/uniquefeature.html (hunk 7 above)
Comment (rajanibusani) [reply javascript/javascript1/week3/uniquefeature.html:20]: yes, changed..

File: javascript/javascript1/week3/uniquefeature.html (hunk 6 above)
Comment (rajanibusani) [reply javascript/javascript1/week3/uniquefeature.html:11]: moved to separate js file..!

File: javascript/javascript1/week3/uniquefeature.html (hunk 8 above)
Comment (rajanibusani) [reply javascript/javascript1/week3/uniquefeature.html]: i removed else condition..

**Pull request merged**